catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/QueryFilterTransformerProvider.java

```diff
@@ -37,6 +40,8 @@
 
   private Map<QName, QueryFilterTransformer> queryFilterTransformerMap = new ConcurrentHashMap<>();
 
+  private Map<String, QName> typeNameQNameMap = new ConcurrentHashMap<>();
```

Comment:
❓ Do we really need the map to be concurrent if everywhere we access it we are synchronized?

Reply:
Switched to use a `HashMap`